features/admin_features/restaurant_owner_creates_restaurant.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Admin can edits a restaurant
+  As an Admin
+  In order to update a restaurant
+  I would like to be able to edit the restaurant
+
+  Background:
+    Given An admin exists "restaurant_owner@example.com" with "password" and super_admin "false"
+    Given the following restaurants with associations exist
+      | name    | address   | description         | res_category_name | menu_name |
+      | My Thai | Stockholm | Some hip thai place | Thai              | Lunch     |
+      | My Mex  | Stockholm | Mexican tacos       | Mexican           | Lunch     |
```

Comment:
Are you setting the owner of these restaurants somehow?

Reply:
just one of them

Reply:
Perhaps you should set the owner in this table?

Reply:
Yeah, we need to do that, but we weren't sure how to put in with FactoryGirl. So we started with an easier solution.

Reply:
Using `find_by` and removing the key from the hash. You've done it before @magnus-thor